src/components/Positioning/InputCard.jsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+const InputCard = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 20px;
+	padding: 5px;
+	border: 1px solid black;
+	list-style: none;
```

Comment:
its a div, are you sure you need a list style here?

Reply:
thank you, i wanted to change this <div> to a <ul> (i have <li> inside) - i correct it now.